config/Dockerfiles/ostree_compose/ostree-compose.sh

```diff
@@ -61,9 +61,12 @@ gpgcheck=0
 skip_if_unavailable=False
 EOF
 
+# Get our latest fedora-atomic-testing.json file and write it to /home/output/logs/
+curl -o /home/output/logs/fedora-atomic-host.json https://pagure.io/fedora-atomic/raw/${branch}/f/fedora-atomic-host.json
```

Comment:
$base_dir/logs not /home/output I believe

Reply:
@johnbieren: I had mimicked the behavior in ostree-image-compoose. I'll update and push again.

Reply:
ostree-image-compose isn't containerized until my PR is merged, this stage already is, hence the discrepancy